packages/safe-core-sdk/src/contracts/GnosisSafe/GnosisSafeContractEthers.ts

```diff
@@ -1,7 +1,12 @@
 import { BigNumber } from '@ethersproject/bignumber'
 import { ContractTransaction } from '@ethersproject/contracts'
 import { SafeTransaction, SafeTransactionData } from '@gnosis.pm/safe-core-sdk-types'
-import { GnosisSafe, GnosisSafeInterface } from '../../../typechain/src/ethers-v5/GnosisSafe'
+import { GnosisSafe as GnosisSafe_V1_1_1 } from '../../../typechain/src/ethers-v5/v1.1.1/GnosisSafe'
```

Comment:
Do you not import relative to `/src` in the code SDK?

Reply:
This will be addresed in a different PR